packages/api/src/platforms/vtex/utils/shouldUpdateShippingData.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  IStoreGeoCoordinates,
+  IStoreSession,
+} from '../../../__generated__/schema'
+import {
+  CheckoutAddress,
+  OrderForm,
+  LogisticsInfo,
+} from '../clients/commerce/types/OrderForm'
+
+export const shouldUpdateShippingData = (
+  orderForm: OrderForm,
+  session: IStoreSession
+) => {
+  if (!hasItems(orderForm)) {
+    return false
+  }
```

Comment:
This should be moved after the `checkGeoCoordinates` according to your tests, right?

Reply:
@icazevedo that make sense but I think it would even make more sense to validate if we have an orderFormId just to already return false if that´s the case and remove this validation. What do you think?

Reply:
MSTM!